src/main/pg/rx.c

```diff
@@ -114,8 +114,18 @@ void pgResetFn_rxConfig(rxConfig_t *rxConfig)
         .sbus_baud_fast = false,
         .msp_override_channels_mask = 0,
         .crsf_use_negotiated_baud = false,
+        .headtracker_max_angle = 0,
+        .headtracker_yaw_shimmy_enable = true,
+        .headtracker_yaw_shimmy_amplitude = 100,
+        .headtracker_yaw_shimmy_count = 5
     );
 
+#ifdef USE_SERIALRX_SBUS
+    rxConfig->serialtx_inverted = 1,
+#else
+    rxConfig->serialtx_inverted = 0,
```

Comment:
Already 0 by default, no need for else clause
```suggestion
```

Reply:
See prior comment. It's now always 0 by detault. Then in `sbusTxInit()` if it is 0 then the UART is inverted but SBus isn't. Hope that makes sense.